lib/kb_blast/Utils/BlastUtil.py

```diff
@@ -1438,7 +1461,10 @@ def parse_BLAST_tab_output (self,
             # we are now making FeatureSets with AMA feature
             #if target_type_name != 'AnnotatedMetagenomeAssembly':  
             if True:
```

Comment:
This looks like cruft that should have been removed.

Reply:
done